sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1003,4 +1007,94 @@ export class ApplicationService extends RecordDataModelService<Application> {
     application.coeStatus = COEStatus.declined;
     return this.repo.save(application);
   }
+
+  /**
+   * Associates an MSFAA number to the application checking
+   * whatever is needed to create a new MSFAA or use an
+   * existing one instead.
+   * @param applicationId application id to receive an MSFAA.
```

Comment:
'an MSFAA' , please change to 'a new MSFAA'

Reply:
It will associate or create, so it will not be necessarily "a new MSFAA" as per the comment right above states "whatever is needed to create a new MSFAA or use an existing one instead.". Does it make sense?

Reply:
its okay understood, please change it to 'a MSFAA'